lib/capistrano/tasks/docker/compose.rake

```diff
@@ -34,7 +34,7 @@ namespace :docker do
       task :build do
         on roles(fetch(:docker_role)) do
           within release_path do
-            execute :"docker-compose", compose_build_command
+            execute :"docker-compose -f #{fetch(docker_compose_path)}", compose_build_command
```

Comment:
@Mossie93 if you can please move the -f option into the `compose_build_command` method, and similiarly in other places as well